js/parts/Interaction.js

```diff
@@ -1186,6 +1205,23 @@ extend(Series.prototype, /** @lends Highcharts.Series.prototype */ {
                         i = i + 1;
                     }
                 }
+
+                // For some types (pie, networkgraph, sankey) opacity is
+                // resolved on a point level
+                if (!series.options.inactiveOtherPoints) {
+                    [
+                        series.group,
+                        series.dataLabelsGroup
```

Comment:
What about the `series.markerGroup`? Are we still looping over every `point.graphic` and setting its opacity? In that case, wouldn't it be more effective to set the series opacity on the marker group?

Reply:
Yes, we still have `marker.states.inactive.something` option. Should we remove that?

Reply:
We must make sure that when we set a whole _series_ to inactive (like a line series), we don't loop over the points and animate the opacity of each. In this case, setting the opacity on the group should be enough.

Reply:
Ok 👍

Reply:
Done